src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 
 {{ figure_markup(
   image="nb_sites_with_cname_tracking.png",
   caption="Websites using CNAME-based tracking on a desktop client",
-  description="Chart showing the number of websites that use a CNAME-based tracker, ordered by the popularity of the tracker.",
+  description="Chart showing the number of websites that use a CNAME-based tracker, ordered by the popularity of the tracker. Pardot is used on 6,619 sites, Adobe Experience Cloud on 2,671, Act-On Software on 869, Oracle Eloqua on 442, Eulerian on 213, Webtrekk on 101, Ingenious Technologies on 40, AT Internet on 34, Tracedock on 28, Wizaly on 15, Criteo on 15, and finally Keyade on 11.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=1887798671&format=interactive",
   sheets_gid="1286114815",
   sql_file="nb_sites_with_cname_tracking.sql"
   )
 }}
 
+{# TODO - this SQL doesn't exist. Also why only desktop? #}
 
-When we look at the rank of the websites that use CNAME-based tracking, we see that 4.2% of the top 1,000 websites embed a CNAME tracker. In the top 10,000, that number falls to 3.33% of websites.
-
+The most popular company performing CNAME-based tracking is <a hreflang="en" href="https://www.pardot.com/">Pardot</a>, which is present on 6,619 desktop websites. Also notable in size is Adobe, with 2,671 publisher domains.
 
+Those numbers may seem a small percentage of our 6.2 million websites crawled on desktop, but that opinion changes when segregating the data by site popularity.
 
 {{ figure_markup(
   image="nb_sites_with_cname_tracking_per_rank.png",
-  caption="Number of websites that use CNAME-based tracking according to the their rank.",
-  description="Chart showing the number of websites that use CNAME-based tracking divided by their popularity rank on the desktop client.",
-  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=231210750&format=interactive",
+  caption="Number of desktop websites that use CNAME-based tracking according to their rank.",
+  description="Chart showing the number of websites that use CNAME-based tracking divided by their popularity rank on the desktop client. Of the top 1,000 sites, 4.20% use CNAME-based tracking, for the top 10,000 it's 3.33%, for top 10,0000 it's 1.18%, for the top million it's 0.29%, and finally for all sites it's 0.09%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=1885483390&format=interactive",
   sheets_gid="518784663",
   sql_file="nb_sites_with_cname_tracking_per_rank.sql"
   )
 }}
 
+{# TODO - this SQL doesn't exist. Also why only desktop? #}
```

Comment:
We are missing the CNAME SQL queries, meaning I couldn't convert to percentage

Reply:
@ydimova

Reply:
Added the CNAME SQL queries to the branch

Reply:
Fixed the linting for these queries as well.